app/src/main/java/by/academy/jc/dubautsova/Anagrams.java

```diff
@@ -0,0 +1,20 @@
+package by.academy.jc.dubautsova;
+
+
+import java.util.Arrays;
+
+import static java.util.Arrays.sort;
+
+public class Anagrams {
+
+    public static String isAnagram(String a, String b){
```

Comment:
Please rework your solution to follow

```
 static boolean isAnagram(String a, String b)
```
signature as was requested by task definition